integration/cloudflare-worker/.npmrc

```diff
@@ -1,2 +1,3 @@
 package-lock=false
-
+audit=false
+fund=false
```

Comment:
I think we can configure these once in the top-level `.npmrc` file

Reply:
No strong objection, but two questions. Does npm inherit config when traversing up the file tree? Also should these be considered independent packages?

Reply:
> Does npm inherit config when traversing up the file tree? 

It does not.